core/user/user.go

```diff
@@ -34,6 +35,11 @@ type User struct {
 	Disabled bool
 }
 
+// UserTag is a tag that refers to a user.
+func UserTag(name string) names.UserTag {
```

Comment:
Tag don't belong in core and we shouldn't be adding any more. We need to look at removing the ones that are already there. Tags are used over the wire; they can be constructed and used in the apiserver facade layers. They should not leak outside that layer. We need to clean up the tech debt we have and doing this dqlite conversion is a great place to start doing that.

Reply:
Hmm, got your position.

And what do we think about pushing UserTag to the domain?  make it a part of the `UserService`?

Reply:
Moved `UserTag` from core to facade layer.